overrides/scripts/hidden.zs

```diff
@@ -201,8 +201,8 @@ var removeAndHide = [
 	<Aroma1997Core:wrenched>,
 	
 	# Chisel
-	<mcheli:lit_drafting_table>,
-	<mcheli:internal>,
+// 	<mcheli:lit_drafting_table>,
+// 	<mcheli:internal>,
```

Comment:
Please remove code instead of commenting it out. Also is there any reason for removing this?